### ISSUE TITLE
pGlyph > fg_layers(asDict=True) crashes FontLab

### ISSUE THREAD

**Issue by alex-phil** [kateliev/TypeRig#61]
Vassil, please take a look at this script:
```
from typerig.proxy.fl.objects.glyph import pGlyph

layers_dict = pGlyph().fg_layers(asDict=True)
print(layers_dict)
```
When you run the script with a glyph selected that has at least one non-empty layer, FontLab 8 (v8244) crashes immediately. When you run the script on a glyph with empty layers only, FontLab 8 does not crash, and the output is correct. Thank you for checking this out.

**Comment by kateliev** [comment]
@alex-phil you are right, but unfortunately this is totally not working. It all looks like a FL8 bug where even indexing the returned list of fg_layers causes the app to crash! The operation fgGlyph.layers[index] is not possible thus crating a dict of it very much impossible also...

 But please indulge my curiosity this .fg_layers() call ... why whoud you need it in first place.. when you have .layers() that will give you back the list of necessary layers to work with. To be honest i do not even remember why i still keep that .fg_layers() because there is absolutely nothing one needs to work with in the FG domain. As a rule of the thumb, you cannot work your way trough it. If you need something done to the FG part of FL, you need to access the FL part of it... convert it to FG... work with it and then covert it back. Otherwise you will crash the app. On the FG layer.. the only thing that is modifiable is Kerning.. and you have to be very careful with it :) So please let me know if you need something special in the FG domain and i might offer you a workaround....
